types/topology.go

```diff
@@ -405,7 +405,7 @@ func (t *Topology) GetNodeMemory(name string) string {
 	return ""
 }
 
-// Return the Sysctl configuration for the given node.
+// GetSysCtl Return the Sysctl configuration for the given node.
```

Comment:
thanks for fixing the comments patter
could you also decapitalise the next word after the method name? `Return->return`

Reply:
done!